src/views/Swap/Swap.vue

```diff
@@ -1074,6 +1077,17 @@ export default {
       this.fromAssetChanged(toAccountId, toAsset, toAmount)
       this.toAssetChanged(fromAccountId, fromAsset)
     },
+    async _getLiqualityLiquidityForAsset() {
```

Comment:
One tricky case might be : 

I am trying to use liquality boost to swap BTC for PUSDC 

It will swap BTC for Matic, then matic for pusdc in 1 inch. 

In current check it won't be a problem because we are checking if the swap proider is 'liquality', but maybe we should check if the  liquality provider is NativeToERC20LiqualityBoost, but then the asset: this.toAsset, should be bridge asset or nativeAsset[ this.toAsset] instead of this.toAsset